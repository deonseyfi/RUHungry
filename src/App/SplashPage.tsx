import React from 'react';
import StartSplash from '../components/StartSplash';

/**
 * Start Splash Page.
 *
 * @returns {React.ReactElement} Splash Page.
 */
const SplashPage = (): React.ReactElement => (
    <StartSplash />
);
export default SplashPage;
