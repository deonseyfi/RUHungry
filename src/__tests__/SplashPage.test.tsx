import React from 'react';
import { mount, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import SplashPage from '../App/SplashPage';

configure({ adapter: new Adapter() });

describe('Splash Page test', () => {
    test('Renders Splash Page', () => {
        const wrapper = mount(<SplashPage />,
        );
        expect(wrapper).toMatchSnapshot();
    });
});
