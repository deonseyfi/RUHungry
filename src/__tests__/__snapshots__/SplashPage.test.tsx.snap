// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash Page test renders Splash Page 1`] = `ReactWrapper {}`;
